### DISCUSSION TITLE
Unable to re-set the email in Account Settings

### DISCUSSION THREAD

**Discussion by Waver-Velvet** [vercel/community#5885] [Account Issue]
### Question

After clicking the "Save" button, it says: 

> The email address [new email] is unavailable.

**Comment by swarnava** [comment]
Hi, it sounds like you have existing email registered with us. In that case, you need to login using the email > Delete the account first before using the same email in different account.

**Reply by Waver-Velvet** [reply]
Thanks. However, when I attempt to login using the email, it says "We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details."

**Comment by pkhlopiachyi** [comment]
I faced same issue today, I sign up with new account, verifed by email with link, but when I'm trying to login, I get such error. New attemp to sign up with this email gives me an error "The email address already exists"
